physics/hierarchical_system_body.hpp

```diff
@@ -58,8 +58,18 @@ HierarchicalSystem<Frame>::ToBarycentric(System& system) {
   auto const semimajor_axis_less_than = [](
       not_null<std::unique_ptr<Subsystem>> const& left,
       not_null<std::unique_ptr<Subsystem>> const& right) -> bool {
-    return left->jacobi_osculating_elements.semimajor_axis <
+    bool const semimajor_axes = left->jacobi_osculating_elements.semimajor_axis &&
```

Comment:
Call this has_semimajor_axes (or have_).  The current identifier feels like it denotes the axis itself.  Same for mean motions.